draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -346,113 +356,138 @@
+			<t>The LSP Burst Window sub-TLV advertises the maximum number of LSPs that the node can receive with no separation interval between LSPs.</t>
+			<t>Type: 1</t>
+			<t>Length: 4 octets</t>
+			<t>Value: number of LSPs that can be sent back to back</t>
+		</section>
+		<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
+			<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
+			<t>Type: 2</t>
+			<t>Length: 4 octets</t>
+			<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the burst window has been used</t>
+			<t>The LSP Transmission Interval is an advertisement of the receiver's steady-state LSP reception rate.</t>
+		</section>
+		<section anchor="LPP" title="LSPs Per PSNP sub-TLV">
+			<t>The sub-TLV LSP Per PSNP (LPP) advertises the number of received LSPs that triggers the immediate sending of a PSNP to acknowledge them.</t>
+			<t>Type: 3</t>
+			<t>Length: 2 octets</t>
+			<t>Value: number of LSPs acknowledged per PSNP</t>
+			<t>A node advertising this sub-TLV with a value LPP MUST send a PSNP once LPP LSPs have been received and need to be acknowledged./t>
 			</section>
-			<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
-				<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
-				<t>Type: 2</t>
-				<t>Length: 4 octets</t>
-				<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the burst window has been used</t>
-				<t>The LSP Transmission Interval is an advertisement of the receiver's steady-state LSP reception rate.</t>
+			<section anchor="Flags" title="Flags sub-TLV">
+				<t>The sub-TLV Flags advertises a set of flags. Flags are standardized and IANA registered.</t>
+				<t>Type: 4</t>
+				<t>Length: Indicates the length in octets (1-8) of the Value field. The length SHOULD be the minimum required to send all bits that are set.</t>
+				<t>Value: List of flags.</t>
+
+				<artwork align="left" pn="Flags">
+          0 1 2 3 4 5 6 7 ...
+         +-+-+-+-+-+-+-+-+...
+         |O|              ...
+         +-+-+-+-+-+-+-+-+...
+				</artwork>
+				<t>When the O flag is set, the LSP will be acknowledged in the order they are received: a PSNP acknowledging N LSPs is acknowledging the N oldest LSPs received. The order inside the PSNP is meaningless. If the sender keeps track of the order to LSP sent, this indication allows a fast detection of loss of LSP. This MUST NOT be used to trigger faster retransmission of LSP. This MAY be used to trigger a congestion signal.</t>
```

Comment:
s/to LSP sent/of LSPs sent/
s/this indication/the ordering/
s/a fast detection of loss of LSP/the fast detection of the loss of an LSP/
s/retransmmission of LSP/retransmission of an LSP/
s/to trigger a congestion signal/as an indication of a congestion event/